app/src/main/java/com/mybus/requirements/AddressValidaror.java

```diff
@@ -0,0 +1,51 @@
+package com.mybus.requirements;
+
+public class AddressValidaror {
+
+    private static final long MAX_NUMBER = 9999;
```

Comment:
Use int instead of long. Also max number should be higher, there are streets with more than 9999 st number, such as Colon, Luro, JB Justo.